fee/src/lib.rs

```diff
@@ -1,19 +1,47 @@
-use {solana_fee_structure::FeeDetails, solana_svm_transaction::svm_message::SVMMessage};
+use {
+    solana_feature_set::{
+        enable_secp256r1_precompile, remove_rounding_in_fee_calculation, FeatureSet,
+    },
+    solana_fee_structure::FeeDetails,
+    solana_svm_transaction::svm_message::SVMMessage,
+};
+
+/// Bools indicating the activation of features relevant
+/// to the fee calculation.
+// DEVELOPER NOTE:
+// This struct may become empty at some point. It is preferable to keep it
+// instead of removing, since fees will naturally be changed via feature-gates
+// in the future. Keeping this struct will help keep things organized.
+#[derive(Copy, Clone)]
+pub struct FeeFeatures {
+    pub remove_rounding_in_fee_calculation: bool,
```

Comment:
do these need to be public, since only this model needs to know which feature(s) matter to fee calculation?